base_ebill_payment_contract/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+
+    _inherit = "res.partner"
+
+    def get_active_contract(self, transmit_method):
+        """Return the active contract for a specific transmit method."""
+        self.ensure_one()
+        # FIXME: Search on non stored field
```

Comment:
You can remove this FIXME if it was for `is_valid`

Reply:
Done